frontend/public/components/app-contents.tsx

```diff
@@ -101,128 +100,548 @@ const pageRouteExtensions = plugins.registry.getRoutePages().map((r) => {
-          <LazyRoute path="/settings/idp/htpasswd" exact loader={() => import('./cluster-settings/htpasswd-idp-form' /* webpackChunkName: "htpasswd-idp-form" */).then(m => m.AddHTPasswdPage)} />
-          <LazyRoute path="/settings/idp/keystone" exact loader={() => import('./cluster-settings/keystone-idp-form' /* webpackChunkName: "keystone-idp-form" */).then(m => m.AddKeystonePage)} />
-          <LazyRoute path="/settings/idp/ldap" exact loader={() => import('./cluster-settings/ldap-idp-form' /* webpackChunkName: "ldap-idp-form" */).then(m => m.AddLDAPPage)} />
-          <LazyRoute path="/settings/idp/oidconnect" exact loader={() => import('./cluster-settings/openid-idp-form' /* webpackChunkName: "openid-idp-form" */).then(m => m.AddOpenIDPage)} />
-          <LazyRoute path="/settings/idp/basicauth" exact loader={() => import('./cluster-settings/basicauth-idp-form' /* webpackChunkName: "basicauth-idp-form" */).then(m => m.AddBasicAuthPage)} />
-          <LazyRoute path="/settings/idp/requestheader" exact loader={() => import('./cluster-settings/request-header-idp-form' /* webpackChunkName: "request-header-idp-form" */).then(m => m.AddRequestHeaderPage)} />
-          <LazyRoute path="/settings/cluster" loader={() => import('./cluster-settings/cluster-settings' /* webpackChunkName: "cluster-settings" */).then(m => m.ClusterSettingsPage)} />
-
-          <LazyRoute path={'/k8s/cluster/storageclasses/~new/form'} exact loader={() => import('./storage-class-form' /* webpackChunkName: "storage-class-form" */).then(m => m.StorageClassForm)} />
-
-          <Route path="/k8s/cluster/:plural" exact component={ResourceListPage} />
-          <LazyRoute path="/k8s/cluster/:plural/~new" exact loader={() => import('./create-yaml' /* webpackChunkName: "create-yaml" */).then(m => m.CreateYAML)} />
-          <Route path="/k8s/cluster/:plural/:name" component={ResourceDetailsPage} />
-          <LazyRoute path="/k8s/ns/:ns/pods/:podName/containers/:name" loader={() => import('./container').then(m => m.ContainersDetailsPage)} />
-          <LazyRoute path="/k8s/ns/:ns/:plural/~new" exact loader={() => import('./create-yaml' /* webpackChunkName: "create-yaml" */).then(m => NamespaceFromURL(m.CreateYAML))} />
-          <Route path="/k8s/ns/:ns/:plural/:name" component={ResourceDetailsPage} />
-          <Route path="/k8s/ns/:ns/:plural" exact component={ResourceListPage} />
-
-          <Route path="/k8s/all-namespaces/:plural" exact component={ResourceListPage} />
-          <Route path="/k8s/all-namespaces/:plural/:name" component={ResourceDetailsPage} />
-
-          <LazyRoute path="/error" exact loader={() => import('./error' /* webpackChunkName: "error" */).then(m => m.ErrorPage)} />
-          <Route path="/" exact component={DefaultPage} />
-
-          <LazyRoute loader={() => import('./error' /* webpackChunkName: "error" */).then(m => m.ErrorPage404)} />
-        </Switch>
+const AppContents = withRouter(
```

Comment:
I'd like to enable prettier for the entire repo, but in the meantime running it on this file makes the PR difficult to review. It's unclear what's changed here.

Reply:
Agreed. We should not be reformatting entire files.

Reply:
My bad. I din't know until raising the PR that the file wasn't prettified. Anyway, I'll redo the changes with the common landing Page